kafka-connect-s3/src/test/java/io/confluent/connect/s3/integration/BaseConnectorNetworkIT.java

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright [2020 - 2020] Confluent Inc.
+ */
+
+package io.confluent.connect.s3.integration;
+
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.s3.model.ListObjectsV2Request;
+import com.amazonaws.services.s3.model.ListObjectsV2Result;
+import com.amazonaws.services.s3.model.S3ObjectSummary;
+import io.confluent.common.utils.IntegrationTest;
+import org.apache.kafka.connect.runtime.AbstractStatus;
+import org.apache.kafka.connect.runtime.rest.entities.ConnectorStateInfo;
+import org.apache.kafka.connect.util.clusters.EmbeddedConnectCluster;
+import org.apache.kafka.test.TestUtils;
+import org.junit.experimental.categories.Category;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
+
+@Category(IntegrationTest.class)
+public abstract class BaseConnectorNetworkIT {
+
+  private static final Logger log = LoggerFactory.getLogger(BaseConnectorNetworkIT.class);
+
+  protected static final long CONSUME_MAX_DURATION_MS = TimeUnit.SECONDS.toMillis(60);
+  protected static final long CONNECTOR_STARTUP_DURATION_MS = TimeUnit.SECONDS.toMillis(1200);
+  protected static final String JSON_FORMAT_CLASS = "io.confluent.connect.s3.format.json.JsonFormat";
+  protected static final String S3_BUCKET = "sink-test-bucket-101";
+
+  protected static AmazonS3 s3RootClient;
+
+  protected static PumbaPauseContainer pumbaPauseContainer;
+
+  protected EmbeddedConnectCluster connect;
+
+  protected void startConnect() throws IOException {
```

Comment:
`startConnect()` is present in `BaseConnectorIT` and can be used in classes the same way by extending that class.